custom/auth/ante/burntax.go

```diff
@@ -47,11 +51,27 @@ func (btfd BurnTaxFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 
 	if !simulate {
 		// Compute taxes again.
-		taxes := FilterMsgAndComputeTax(ctx, btfd.treasuryKeeper, msgs...)
+		taxes := FilterMsgAndComputeTax(ctx, btfd.TreasuryKeeper, msgs...)
 
 		// Record tax proceeds
 		if !taxes.IsZero() {
-			err = btfd.bankKeeper.SendCoinsFromModuleToModule(ctx, types.FeeCollectorName, treasury.BurnModuleName, taxes)
+			if currHeight >= TaxPowerSplitHeight {
```

Comment:
Why we don't remove the `fork height` logic and use normal upgrade handler for ship this feature ?

Reply:
Lets resolve this after we are done with the overall upgrade of the chain. I think we need to wait for @inon-man research to complete before we know when that can happen, so it will be wiser to do this in a later PR and merge in the code as is :)